iFixit/react-commerce
# Remove product list hero image

fixes #269 

## Changelog

- Remove hero image from product list

## QA

1. Visit the Vercel PR preview (replace `vercel.app` with `cominor.com`)
2. Visit product list pages and verify that the hero does not have an image
